src/lib.rs

```diff
@@ -40,15 +40,51 @@ static mut GC_ALLOCATOR: GCMalloc = GCMalloc;
 
 static mut COLLECTOR: Option<Collector> = None;
 
+/// A garbage collected pointer. 'Gc' stands for 'Garbage collected'.
 #[derive(Debug)]
 pub struct Gc<T> {
     objptr: *mut T,
 }
 
+/// A garbage collected value is stored with a 1 machine word sized header. This
```

Comment:
I assume this word has a specific bit pattern? Can we document the layout?

Reply:
Sure: c484ee07086a94b97ecbe648796e987923e78155